tests/Form/FormTest.php

```
@@ -0,0 +1,1133 @@
+                ->enctype(Enctype::MULTIPART_FORM_DATA)
+                ->render(),
+            "Failed asserting that element renders correctly with 'enctype' attribute.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(
+            Form::class,
+            ['class' => 'default-class'],
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)